programmers/20210722/프린터.js

```diff
@@ -0,0 +1,21 @@
+function solution(priorities, location) {
+    const idx = [];
+    let order = 0;
+    let print;
```

Comment:
```suggestion
    let print = 0; 
```
자료형을 알 수 있도록 값을 할당해주는것이 좋을 것 같습니다

Reply:
배워갑니다